src/Command/Diff.php

```diff
@@ -86,25 +86,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $handler = new DiffHandler($envName, $cwd, $output->isDecorated());
+        $errorIo = $this->getErrIo($input, $output);
```

Comment:
Ah.. Because it stays good enough for an internal so I thought it's not worth it. But if you think it makes sense I can update it (I would not want to hurt you :trollface:)